docs/4.3/metrics_logs_reference.md

```diff
@@ -0,0 +1,99 @@
+# Metrics
+
+## Teleport Prometheus Endpoint
+
+Teleport provides HTTP endpoints for monitoring purposes. They are disabled
+by default, but you can enable them:
```

Comment:
"enable them:" -> "enable them using the `--diag-addr` flag to `teleport start`:"